lib/src/rules/unnecessary_overrides.dart

```diff
@@ -227,13 +235,22 @@ class _UnnecessaryMethodOverrideVisitor
   _UnnecessaryMethodOverrideVisitor(super.rule);
 
   @override
-  ExecutableElement? getInheritedElement(node) => node.lookUpInheritedMethod();
+  ExecutableElement? getInheritedElement(node) {
+    var inheritedMethod = node.lookUpInheritedMethod();
+    if (inheritedMethod == null) return null;
+    var enclosingTypeAsInstanceOfInheritedEnclosingType =
```

Comment:
I am very open to better names here, both for the local variable and for the extension method on MethodDeclaration :D. I thought for a while though and couldn't think of anything shorter which satisfied me.